www/templates/control/main-control.html

```diff
@@ -1,29 +1,14 @@
 <ion-view view-title="Profile" ng-class="ionViewBackgroundClass()">
-  <!-- <div align-title="center" class="bar bar-positive bar-subheader">
-      <div class="buttons" style="text-align:center">
-        <button class="button button-assertive icon-right ion-stop"
-            ng-click="forceTransition('STOP_TRACKING')"> Stop Tracking </button>
-        <button class="button icon-right ion-refresh" ng-click="refreshScreen()"/> </button>
-        <button class="button icon-right ion-cloud" ng-click="forceSync()"> Sync </button>
-        <button class="button button-balanced icon-right ion-play"
-            ng-click="forceTransition('START_TRACKING')"> Start Tracking </button>
-      </div>
-  </div> -->
-
   <ion-content>
-<!--     <div style="text-align: center;">
-      <img src="img/avatar_1.png" ng-style="getAvatarStyle()">
-    </div> -->
-
     <div class="control-list-item">
       <div class="control-list-text">{{settings.auth.email}}</div>
-      <div ng-click="returnToIntro()" ng-style="getIconButtonStyle('E0A1A1')"><i class="icon ion-log-out" ng-style="getIconStyle()"></i></div>
+      <div ng-click="returnToIntro()" id ="signout" class="icon-button-style"><i class="icon ion-log-out"></i></div>
```

Comment:
not sure that this should be called "signout", because it really doesn't sign you out. If you click on this and then kill the app and then restart it, for example, I think that the prior login will still work.